bin/pymultimonaprs

```diff
@@ -23,7 +23,7 @@ args = parser.parse_args()
 config = json.load(open(args.config))
 
 logger = logging.getLogger('pymultimonaprs')
-loglevel = logging.DEBUG if args.verbose else logging.INFO
+loglevel = logging.DEBUG if (int(config['rtl']['debug'] == 1)) else logging.INFO
```

Comment:
You are creating a bool value, converting it to int and then let python autocast it back to bool.
Also you make pymultimonaprs ignore the --verbose cli argument but don't remove the cli-parser option. But I wouldn't remove that option - insted it should use debug mode if called with `--verbose` or if `verbose = true` in config.

Also you can't just expect that existing config files (installations that get this change via pkg update) have the `verbose` key. So you need to use the `dict.get(dict_key, failover_value)` function.

So the line would look like:
```python
loglevel = logging.DEBUG if args.verbose or config.get('verbose', False) else logging.INFO
```